website/docs/r/project_api_key.html.markdown

```diff
@@ -57,24 +57,19 @@ resource "mongodbatlas_project_api_key" "test" {
 
 ~> **NOTE:** Project created by API Keys must belong to an existing organization.
 
-# project_assignment
+### project_assignment
 Project Assignment attribute is optional (Use project_assignment going forward as role_names parameter above is deprecated)
 
 * `project_id` - (Required) Project ID to assign to Access Key
-* `role_names` - Name of the role. This resource returns all the roles the user has in Atlas.
+* `role_names` - (Required) List of Project roles that the Programmatic API key needs to have. Ensure you provide: at least one role and ensure all roles are valid for the Project.  You must specify an array even if you are only associating a single role with the Programmatic API key.
 The following are valid roles:
   * `GROUP_OWNER`
   * `GROUP_READ_ONLY`
   * `GROUP_DATA_ACCESS_ADMIN`
   * `GROUP_DATA_ACCESS_READ_WRITE`
   * `GROUP_DATA_ACCESS_READ_ONLY`
-  * `GROUP_CLUSTER_MANAGER`  
-  * 
-### Programmatic API Keys
```

Comment:
This section is unrelated to project api key resource, seems like a copy paste that came from project resource documentation